index.html

```diff
@@ -917,6 +918,7 @@ <h4>Box Model</h4>
               <a data-cite="CSS-WRITING-MODES-3#line-under"><dfn>line-under</dfn></a>.
             </li>
           </ul>
+          </p>
```

Comment:
I think you (correctly) renoved this /p in the other Pr this is bringing it back. I fixes the extra span ans re-checked with validator.nu thanks for the headsup.